db/migrate/20210322225751_create_reminder_templates.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReminderTemplates < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reminder_templates do |t|
+      t.integer :experiment_group, null: false
+      t.string :message, null: true
+      t.integer :appointment_offset, null: true
```

Comment:
Does `null: true` make sense here?

Reply:
I believe it does because experiments for non-active patients will not use an offset. They will just be scheduled over the next 30 days.

Reply:
> They will just be scheduled over the next 30 days.

Scheduled how? Is there an offset hidden there we could capture here?

Reply:
I just checked the original PRD, and I'm mistaken. They should use the offset, because some of them will get a cascade of messages. Assuming that I also make ExperimentGroup an AR model, this should be required.